Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -517,22 +532,28 @@
   name: chief engineer PDA
   description: Looks like it's barely been used.
   components:
   - type: Pda
     id: CEIDCard
     state: pda-ce
+    penSlot:
+      startingItem: LPPCEPen
+      priority: -1
+      whitelist:
+        tags:
+        - Write
```

yamllint

[error] wrong indentation: expected 10 but found 8

(indentation)